generic/nginx/security/missing-internal.yaml

```diff
@@ -23,7 +23,7 @@ rules:
   - generic
   severity: WARNING
   message: >-
-    This location block contains a 'proxy_pass' directive but does not contain the 'internal'
```

Comment:
I don't see `$URL` defined in any of the patterns above. Did you mean to include this?

Reply:
nope, this is "debugging" information :laughing: 
thanks!